examples/zone-printer/src/main.go

```diff
@@ -38,13 +56,74 @@ func main() {
 		log.Printf("info: determined zone: %q", zone)
 	}
 
-	log.Println("starting to listen on port 80")
-	http.HandleFunc("/", handle)
-	err := http.ListenAndServe(":80", nil)
-	log.Fatal(err)
+	finished := make(chan bool)
+
+	log.Printf("starting to listen on port %d", mainPort)
+	mainServer := http.NewServeMux()
+	mainServer.HandleFunc("/", handlePing)
```

Comment:
please point root to `handleLocation` handler. this app is supposed to print zone mainly. anything else is secondary.

Reply:
Done